rally-domain/src/main/groovy/yakworks/rally/tag/model/Tag.groovy

```diff
@@ -31,11 +31,10 @@ class Tag implements NameCodeDescription, RepoEntity<Tag>, Serializable {
     }
 
     static constraintsMap = [
-        description:[ description: "The description for this tag",
-            nullable: true, maxSize: 255],
-        name:[ description: "The tag name",
-            nullable: false, maxSize: 50, unique: ["entityName"] ],
-        entityName:[description: "The entity this tag can be applied to. May be a comma sep list of entity names", nullable: true],
+        name:[ d: 'The name of tag', nullable: false, maxSize: 255],
+        description:[ description: "The description for this tag", nullable: true, maxSize: 255],
+        name:[ description: "The tag name", nullable: false, unique: ["entityName"],  maxSize: 50],
```

Comment:
name is here twice. also stick with the shorter d: here, same as some above. either do all d: or all description:

Reply:
Fixed